framework/wazuh/core/configuration.py

```diff
@@ -401,16 +401,15 @@ def _rootkit_files2json(filepath):
 
     # file_name ! Name ::Link to it
     regex_comment = re.compile(r"^\s*#")
-    regex_check = re.compile(r"^\s*(.+)\s+!\s*(.+)\s*::\s*(.+)")
+    regex_check = re.compile(r"^\s?(.+)\s!\s?(.+)\s?::\s?(.+)")
```

Comment:
In lines 413-414, strips to each group are made, so we could avoid using \s in this regex:

```suggestion
    regex_check = re.compile(r"^(.+)!(.+)::(.+)")
```

Reply:
Also check this in the regex defined in lines 434, 435